ios/RNSScreenStack.mm

```diff
@@ -1117,9 +1117,8 @@ - (void)mountChildComponentView:(UIView<RCTComponentViewProtocol> *)childCompone
 
   [_reactSubviews insertObject:(RNSScreenView *)childComponentView atIndex:index];
   ((RNSScreenView *)childComponentView).reactSuperview = self;
-  dispatch_async(dispatch_get_main_queue(), ^{
-    [self maybeAddToParentAndUpdateContainer];
-  });
+  // Child update operation is performed in the mountingTransactionDidMount method to prevent error, while
+  // navigating to `n` different screens at the same time.
```

Comment:
I think this comment isn't very helpful outside the scope of this PR since the deleted lines won't be visible. WDYT?

Reply:
I just made it for the sake of questions like "why we cannot move this update method to the mount and unmount methods, if they're doing the same?" (especially, that removing this method won't be visible in git blame), but if you think this comment is unnecessary, I can remove it.